# HG changeset patch
# User Rik <rik@octave.org>
# Date 1512614971 28800
#      Wed Dec 06 18:49:31 2017 -0800
# Node ID 0755de78071ca2062ef04958bc5a8d62a4057d3d
# Parent  2e525915385749f1d59d6c113d308a65da8e988f
Fix quoting error from cset b1d1229d9e83.

* __publish_html_output__.m: Match starting single quote with an ending single
quote.

diff --git a/scripts/general/private/__publish_html_output__.m b/scripts/general/private/__publish_html_output__.m
--- a/scripts/general/private/__publish_html_output__.m
+++ b/scripts/general/private/__publish_html_output__.m
@@ -120,17 +120,17 @@ function outstr = do_header (title_str, 
  'config=TeX-MML-AM_CHTML"></script>']);
 
   stylesheet_str = sprintf ("%s\n",
 "<style>",
 "body > * {",
 "  max-width: 42em;",
 "}",
 "body {",
-'  font-family: "Roboto Condensed", sans-serif;",
+'  font-family: "Roboto Condensed", sans-serif;',
 "  padding-left: 7.5em;",
 "  padding-right: 7.5em;",
 "}",
 "pre, code {",
 "  max-width: 50em;",
 "  font-family: monospace;",
 "}",
 "pre.oct-code {",
@@ -152,17 +152,17 @@ function outstr = do_header (title_str, 
 "footer {",
 "  margin-top: 2em;",
 "  font-size: 80%;",
 "}",
 "a, a:visited {",
 "  color: Blue;",
 "}",
 "h2 {",
-'  font-family: "Roboto Condensed", serif;",
+'  font-family: "Roboto Condensed", serif;',
 "  margin-top: 1.5em;",
 "}",
 "h2 a, h2 a:visited {",
 "  color: Black;",
 "}",
 "</style>");
 
   outstr = sprintf ("%s\n",
